packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -202,15 +202,24 @@ portfolio.addProduct(product);
 
 TagOptions allow administrators to easily manage tags on provisioned products by creating a selection of tags for end users to choose from.
 For example, an end user can choose an `ec2` for the instance type size.
-TagOptions are created by specifying a key with a selection of values.
+TagOptions are created by specifying a key with a selection of values, 
```

Comment:
No comma since following clause not independent

Reply:
wow.

Truthfully did it to split up lines to fit within max.  will try to fix.